kitsune/wiki/views.py

```diff
@@ -537,6 +537,22 @@ def edit_document(request, document_slug, revision_id=None):
         'locked_by': locked_by})
 
 
+@csrf_exempt
+@login_required
+@require_POST
+def draft_revision(request):
+    """Create a Draft Revision.
+
+    User can have only one draft revision for a translated document. Store the draft with
+    parent document, user and locale. Get the parent document from the based on revision"""
+    draft_form = DraftRevisionForm(request.POST)
+    if draft_form.is_valid():
+        _create_draft_revision(request, draft_form)
+        return HttpResponse('')
```

Comment:
Better would be `return HttpResponse(status=204)`. [`HTTP 204`](http://http.cat/204) is "No Content" and is appropriate to use in this context.